etc/profilicate.py

```diff
@@ -63,7 +63,7 @@ def contents(self):
 
 tid = 0
 threads = []
-for (name, raw_samples) in sorted(thread_data.iteritems(), key=lambda x: thread_order[x[0]]):
+for (name, raw_samples) in sorted(iter(thread_data.items()), key=lambda x: thread_order[x[0]]):
```

Comment:
I think the `iter()` is is not needed in this case, since `sorted` can consume iterables directly and `items()` returns view objects that are iterable.